docs/jitserver.md

```diff
@@ -0,0 +1,88 @@
+<!--
+* Copyright (c) 2017, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# JITServer technology (technical preview)
+
+![Start of content that applies to Java 8 and later](cr/java8.png)
+![Start of content that applies to Java 11 and later](cr/java11.png)  
+**Linux&reg; on x86-64 only**
+
+JITServer technology decouples the JIT compiler from the VM and lets the JIT compiler run remotely in its own process. This prevents your Java&trade; application suffering possible negative effects due to CPU and memory consumption caused by JIT compilation.
+
+This technology can potentially improve quality of service, robustness, and even performance of Java applications. We recommend trying this technology if the following criteria are met:
+
+- Your Java application is required to compile many methods using JIT in a relatively short time.
+- The application is running in an enviroment with limited CPU or memory, which can worsen interference from the JIT compiler.
+- The network latency between JITServer and client VM is relatively low.
+
+For more details about JITServer technology, its pros and cons, and when best to use it, see the blog [Free your JVM from the JIT with JITServer Technology](https://blog.openj9.org/2020/01/09/free-your-jvm-from-the-jit-with-jitserver-technology/).
+
+## Using JITServer technology
+
+OpenJ9 builds available from [AdoptOpenJDK](https://adoptopenjdk.net/archive.html?variant=openjdk11&jvmVariant=openj9) have JITServer technology included, though you must explicitly invoke it.
```

Comment:
Might be better to avoid talking about the AdoptOpenJDK binaries here and just say that it is not enabled by default. The topic is qualified as Linux on x86-64 and we know that all the related binaries at Adopt are building it in.